src/rgw/rgw_sal_motr.cc

```diff
@@ -357,7 +364,7 @@ int MotrUser::list_buckets(const DoutPrefixProvider *dpp, const string& marker,
     ent.decode(iter);
 
     std::time_t ctime = ceph::real_clock::to_time_t(ent.creation_time);
-    ldpp_dout(dpp, 20) <<__func__<< "got creation time: " << std::put_time(std::localtime(&ctime), "%F %T") << dendl;
+    ldpp_dout(dpp, 20) <<__func__ << "got creation time: " << std::put_time(std::localtime(&ctime), "%F %T") << dendl;
```

Comment:
log level macro not used here

Reply:
As this is the reference PR, I have added log level macro only in versioning and tagging log statements. This change (line no 352) is related to another feature, hence I didn't make macro changes there.

Reply:
got it, yeah as discussed, we will avoid touching all features at a time, specially when we dont know which all logs would make sense and will allow feature owners to revisit their logs and update appropriate log levels with macros